.travis.yml

```diff
@@ -14,6 +14,7 @@ rvm:
 env:
   global:
     - JRUBY_OPTS='--debug -J-Xmx1000M' # get more accurate coverage data in JRuby
+    - TEST_QUEUE_WORKERS=2
```

Comment:
Please add some note here explaining the need for this for future reference.

Reply:
I've added a comment for this. Sorry too late response.